packages/config/config/devices/0x016a/ft112-k.json

```diff
@@ -95,15 +95,15 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Basic Set",
+					"label": "Basic set",
 					"value": 1
 				},
 				{
 					"label": "Sensor binary report",
 					"value": 2
 				},
 				{
-					"label": "Basic Set and sensor binary report",
+					"label": "Basic set and sensor binary report",
```

Comment:
One could argue that these are the CC names, so `Basic Set` and `Sensor Binary Report` would be better. In any case, they should be consistent.